css/easyddb_legend.css

```diff
@@ -55,11 +60,40 @@ div.pane-ding-availability-legend .availability-legend {
 }
 
 div.pane-ding-availability-legend .availability-legend .availability-legend-item {
-  margin: 0 10px 10px 0;
+  margin: 0 8px 0 0;
 }
 
 div.pane-ding-availability-legend .availability-legend .availability-legend-item .availability-label {
-  color: #fff;
   font-weight: 400;
   font-size: 100%;
 }
+
+.field-type-ting-type li,
+.js-search-result--availability-link {
+  border-radius: 5px;
+  padding: 5px 11px;
+  color: white !important;
+  display: inline-block;
+  text-decoration: underline;
+  margin: 2px 0 !important;
+}
+
+.availability.available {
```

Comment:
Shouldn't you use the existing definitions, instead of creating new ones?

Reply:
It is not really possible for the current state of ding_availability.

Reply:
What do you mean? You can just add `.availability.available,` to L25.

Reply:
Yes, you are right. There is no sense to duplicate same definitions.